### DISCUSSION TITLE
VS Code runs once only

### DISCUSSION THREAD

**Discussion by ali19833** [microsoft/vscode-discussions#398] [Extension Development QnA] [answered]
Hello Ladies and Gentleman,

my question is quite simple and is regarding VSCode (obviously). I have the extention "code runner" in VSCode and I run my code either by code runner or use VSCode run-button. The problem is VSCode runs the code once only. When I modify the code and try to re-run the code again it doesn't show anything in the terminal. I need to close the terminal window first in order to get it run again.

I use the Python as a programming language. If any more information is needed, please let me know.

Thank you a lot

**Comment by burkeholland** [comment, marked as answer]
Thank you for your question. This discussion board is for extension development, and this question is probably better answered via the CodeRunner GitHub Issues or StackOverflow.

Closing this in favor of the above options. If you believe this is a mistake, please feel free to re-open.

**Comment by ali19833** [comment]
thank you i will adress this question in the mentioned website.

Cheers
